views/extras.tmpl

```diff
@@ -225,22 +227,36 @@
             $("#decode_header").text("Sent tx");
             $("#decoded_tx").text(evt);
         })
+
         var updateBlockData = function (event) {
             console.log("Received newblock message", event);
             var newBlock = JSON.parse(event);
-            var confirms = $('[data-confirmation-block-height]');
-            $.each(confirms, function(i, v) {
-                if (isNaN($(v).text())) {
-                    $(v).text("0");
-                }
-                var confirmations = newBlock.block.height - $(v).data('confirmation-block-height');
-                $(v).text("(" + confirmations + (confirmations > 1? " confirmations": " confirmation") + ")")
-            })
 
             // block summary data
             var b = newBlock.block;
             desktopNotifyNewBlock(b);
 
+            var currentTxHash = $('#spanTxID').data('tx-page-txid');
+            if(currentTxHash) { // we are on the tx page
+                Turbolinks.reload()
+                /*if((b.Tx.filter(function (tx) {
+                    return tx.TxID === currentTxHash
+                })).length > 0){
+                    // the current tx is in the received block
+                    Turbolinks.visit(window.location)
+                }*/
+            }else {
```

Comment:
I think our JS style is a little wonky.  @gozart1 What style guide would you recommend we follow?

Reply:
https://crockford.com/javascript/code.html
Is the oldest, simplest, most common I know of

Reply:
Since there is a lot of edge js starting to be used in our code, we might consider using a compiler like https://babeljs.io/ 
Which would let us go a futuristic as we want while supporting old browsers, and have all sorts of other benefits from code-minification, to enforcing a particular formatting style.

The main reason I haven't suggested it is because it adds a bit of overhead to the dev env. But it's not too bad.
Can open an issue of it, it there's interest.